sechub-integrationtest/src/test/java/com/daimler/sechub/integrationtest/scenario12/PDSWebScanJobScenario12IntTest.java

```diff
@@ -37,43 +40,59 @@ public void pds_web_scan_has_expected_info_finding_with_given_target_url_and_pro
+        //
         // look at 'integrationtest-webscan.sh' for implementation details
         // finding 1: contains target url and more 
         // finding 2: contains sechub configuration (only web parts)
-        String pdsScanConfigurationJSON = assertReport(sechubReport).
+        String descriptionFinding2WithDataInside = assertReport(sechubReport).
             finding(0).
                 hasSeverity(Severity.INFO).
                 hasDescriptionContaining("PRODUCT2_LEVEL=4711").// this comes from custom mandatory parameter from PDS config
-                hasDescriptionContaining("PDS_SCAN_TARGET_URL="+targetURL). // this is a default generated parameter which will always be sent by SecHub without being defined in PDS config!
+                hasDescriptionContaining("PDS_SCAN_TARGET_URL=" + targetURL). // this is a default generated parameter which will always be sent by SecHub without being defined in PDS config!
             finding(1).
                 hasDescriptionContaining("PDS_SCAN_CONFIGURATION={").
                 getDescription();
-        pdsScanConfigurationJSON=pdsScanConfigurationJSON.substring("PDS_SCAN_CONFIGURATION=".length());
-        
-        // the returned JSON must be a valid sechub scan configuration
-        SecHubScanConfiguration configuration = SecHubScanConfiguration.createFromJSON(pdsScanConfigurationJSON);
-        assertEquals("ProjectId not as expected", project.getProjectId(), configuration.getProjectId());
-        assertFalse(targetURL, configuration.getCodeScan().isPresent());
-        assertFalse(targetURL, configuration.getInfraScan().isPresent());
-        assertTrue(targetURL, configuration.getWebScan().isPresent());
         
-        SecHubWebScanConfiguration webConfiguraiton = configuration.getWebScan().get();
-        assertFalse(webConfiguraiton.getUris().isEmpty());
+        String returndPdsScanConfigurationJSON = 
+                 descriptionFinding2WithDataInside.substring("PDS_SCAN_CONFIGURATION=".length());
         /* @formatter:on */
+
+        // the returned JSON must be a valid sechub scan configuration
+        SecHubScanConfiguration returnedConfiguration = SecHubScanConfiguration.createFromJSON(returndPdsScanConfigurationJSON);
+        assertEquals("ProjectId not as expected", project.getProjectId(), returnedConfiguration.getProjectId());
+        assertFalse(targetURL, returnedConfiguration.getCodeScan().isPresent());
+        assertFalse(targetURL, returnedConfiguration.getInfraScan().isPresent());
+        assertTrue(targetURL, returnedConfiguration.getWebScan().isPresent());
+
+        SecHubWebScanConfiguration webConfiguration = returnedConfiguration.getWebScan().get();
+        assertNotNull(webConfiguration.getUri());
+        assertEquals(JSONConverter.get().toJSON(configuration, true), JSONConverter.get().toJSON(returnedConfiguration, true));
```

Comment:
Please add some excludes/includes checks here as well
inside
`sechub-integrationtest-webscanconfig-all-options.json`
you already defined includes and excludes.
Please check that the web configuration returned by Test PDS contains all of these includes and excludes so we test a PDS script will have the information available at runtime.

Reply:
`sechub-integrationtest-webscanconfig-all-options.json` already contains includes and excludes. As a result, the test ensures, that the includes/excludes are transferred to the PDS and back.